templates/abilities.html

```diff
@@ -167,14 +167,38 @@ <h2>Abilities</h2>
                                     </div>
                                 </div>
                             </div>
+                            <div class="field is-horizontal">
+                                <div class="field-label is-small">
+                                    <label class="label">
+                                        <span class="icon is-small"><em class="fas fa-search"></em></span>
+                                    </label>
+                                </div>
+                                <div class="field-body">
+                                    <div class="field">
+                                        <div class="control auto-complete" x-data="{focusSearchResults: true}">
```

Comment:
We can remove the search from this modal since there's a search on the page for it.